solidity/random-beacon/contracts/RandomBeaconGovernance.sol

```diff
@@ -760,22 +801,22 @@ contract RandomBeaconGovernance is Ownable {
         return
             getRemainingChangeTime(
                 dkgResultChallengePeriodLengthChangeInitiated,
-                24 hours
+                STANDARD_PARAMETER_GOVERNANCE_DELAY
             );
     }
 
     /// @notice Get the time remaining until the DKG submission eligibility
```

Comment:
"Get the time remaining until the DKG submission eligibility" => " Get the time remaining until the DKG result submission eligibility"